modular_citadel/code/game/objects/items/melee/eutactic_blades.dm

```diff
@@ -269,6 +270,7 @@
 	unwieldsound = 'sound/weapons/neboff.ogg'
 	hitsound = "swing_hit"
 	armour_penetration = 60
+	slowdown = 1
```

Comment:
This doesn't take into consideration wheter the swordis on or off anymore.
The leftover var change on line 310/312 is proof it was supposed to have a different slowdown on wield.